### PR TITLE
release: bump to v0.1.1

### PR DESCRIPTION
**Pull request by skyzh**
Signed-off-by: Alex Chi <iskyzh@gmail.com>

I've already published risinglight_proto.

The final problem is sqlparser-rs. Shall we publish it as `risinglight_sqlparser`?

ref https://github.com/risinglightdb/risinglight/issues/574

### CODE REVIEW

File: proto/Cargo.toml [hunk 1]
```diff
@@ -1,7 +1,13 @@
 [package]
 name = "risinglight_proto"
-version = "0.1.0"
+version = "0.1.1"
 edition = "2021"
+description = "Proto for RisingLight"
+license = "MIT OR Apache-2.0"
```
Comment (xxchan) [inline proto/Cargo.toml:6]: What's this? 🤯

**Review by xxchan — APPROVED** [review]

File: proto/Cargo.toml (hunk 1 above)
Comment (skyzh) [reply proto/Cargo.toml:6]: The protobuf definition for storage :)

Comment (xxchan) [reply proto/Cargo.toml:6]: I mean the license

Comment (skyzh) [reply proto/Cargo.toml:6]: Copied from sqllogictest :)

**Pull request merged**